src/sentry/incidents/endpoints/project_alert_rule_details.py

```diff
@@ -25,8 +28,23 @@ def get(self, request: Request, project, alert_rule) -> Response:
         ``````````````````
         :auth: required
         """
-        data = serialize(alert_rule, request.user, AlertRuleSerializer())
-        return Response(data)
+        serialized_alert_rule = serialize(alert_rule, request.user, AlertRuleSerializer())
```

Comment:
@roggenkemper Maybe we should make this part of the serializer and make it an option so by default we don't serialize with it?